db/migrations/admin-migrations/2024-05-06.0-alter-tenants-add-add-distribution-account-type-and-status.sql

```diff
@@ -0,0 +1,27 @@
+-- +migrate Up
+
+CREATE TYPE distribution_account_type AS ENUM (
+    'ENV_STELLAR',
+    'DB_VAULT_STELLAR',
+    'DB_VAULT_CIRCLE'
+);
+CREATE TYPE distribution_account_status AS ENUM (
+    'ACTIVE',
+    'PENDING_USER_ACTIVATION'
```

Comment:
so right now, the default is to instantiate the account in the `ACTIVE` state, do we plan to change this? and if so, are we going to allow the user to activate the account through some endpoint?

Reply:
It starts in ACTIVE so this is not a breaking change, but this is being explicitly set in the API call.

[This ticket](https://stellarorg.atlassian.net/browse/SDP-1167) should make the configuration explicit. For instance, Stellar is starting as ACTIVE as it should, but for Circle it won't start as active